Titanium.Web.Proxy/RequestHandler.cs

```diff
@@ -169,6 +189,8 @@ partial class ProxyServer
         //So for HTTPS requests we would start SSL negotiation right away without expecting a CONNECT request from client
         private async Task HandleClient(TransparentProxyEndPoint endPoint, TcpClient tcpClient)
         {
+            var processId = GetProcessIdFromPort(((IPEndPoint)tcpClient.Client.RemoteEndPoint).Port, endPoint.IpV6Enabled);

```

Comment:
I liked the way it was before. It should be something lazily evaluated to avoid performance overhead especially when user don't care about process Id. Also process Id is invalid when client request is originating from a different machine, in that case its not even worth to call TcpTable function.